recipe/meta.yaml

```diff
@@ -88,7 +88,8 @@ outputs:
         # CuPy default detects CUDA from nvcc, but on Conda-Forge's dockers nvcc lives in a different place...
         # With conda-forge/nvcc-feedstock#58, CUDA_PATH is set correctly
         - export NVCC="$(which nvcc)"                                                # [linux]
-        - export CUPY_NUM_BUILD_JOBS=1                                               # [aarch64 or ppc64le or (linux64 and cuda_compiler == "nvcc")]
+        - export CUPY_NUM_NVCC_THREADS=1                                             # [win]
```

Comment:
Was looking at [the build environment variables]( https://docs.cupy.dev/en/stable/reference/environment.html#for-installation ) and noticed that in addition to build jobs, CuPy configures NVCC to do parallel compilation. This is configured with [`CUPY_NUM_NVCC_THREADS`, which defaults to 2]( https://docs.cupy.dev/en/stable/reference/environment.html#envvar-CUPY_NUM_NVCC_THREADS )

Went ahead and set this to `1` for Windows

That said, Windows is still seeing memory issues. This should at least help some